libraries/botbuilder-ai-orchestrator/src/orchestratorAdaptiveRecognizer.ts

```diff
@@ -84,16 +109,26 @@ export class OrchestratorAdaptiveRecognizer
      */
     public externalEntityRecognizer: Recognizer;
 
+    /**
+     * Enable entity detection if entity model exists inside modelFolder. Defaults to false.
+     */
+    public scoreEntities: boolean = false;
```

Comment:
You can omit `: boolean`. You are defaulting this to `false` so the compiler can infer the type.